test/functionalTest/cases/1187_empty_attribute_in_create/empty_attribute_in_create.test

```diff
@@ -120,11 +120,6 @@ Date: REGEX(.*)
         "type": null,
         "value": ""
     },
-    "A2": {
```

Comment:
Actually, there has been a bug fix.
We talked about this ...
Before, the broker signaled error and created A2 (in the previous step).
Now, the same error is signaled but A2 is not created.

This is better than before.

Even better would be to NOT signal error and to create A2, but that is a bigger fix and not included in this PR.